tomcat-gcp-lib/pom.xml

```diff
@@ -90,7 +90,7 @@
         <version>2.20</version>
         <configuration>
           <excludes>
-            <exclude>**/*IntegrationTest.java</exclude>
+            <exclude>**/*IT.java</exclude>
```

Comment:
You don't need to exclude `**/*IT.java` because it's excluded [by default](http://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html).